generator/main.go

```diff
@@ -33,7 +33,9 @@ func generateConfig(nodes *Node, nameToNode map[string]*Node) {
 		log.Infof("Generated %d metrics for module %s", len(outputConfig[name].Metrics), name)
 	}
 
+	config.DoNotHideSecrets = true
 	out, err := yaml.Marshal(outputConfig)
+	config.DoNotHideSecrets = false
```

Comment:
There isn't, however I'm playing it safe as future changes might use it in another way and we don't want the secrets accidentally exposed.